packages/fxa-payments-server/src/lib/types.tsx

```diff
@@ -12,3 +12,9 @@ export interface GenericObject {
 
 // https://stackoverflow.com/questions/48215950/exclude-property-from-type
 export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+
+// Relaxed type derived from a Function where we ignore the return value
+// in actual use. Makes it easier to write mocks.
+export type FunctionWithIgnoredReturn<T extends (...args: any) => any> = (
```

Comment:
This is a generic type I'm trying out so that we can be a *bit* more specific than just `Function` in component props, yet not so specific that we can't easily write mocks in types & stories.